build.gradle

```diff
@@ -69,6 +69,18 @@ task installRealmJava(type:Task) {
     description = 'Install the Realm library and Gradle plugin into mavenLocal()'
 }
 
+task installRobolectric(type:GradleBuild) {
+    description = 'Install the Robolectric library into example'
```

Comment:
@zaki50 If you set `coreSourcePath` on `realm.properties`, `assembleRobolectric` will use this source path instead of the installed Core lib.